core/src/main/java/org/axonframework/eventsourcing/eventstore/EmbeddedEventStore.java

```diff
@@ -53,8 +62,11 @@
  * @since 3.0
  */
 public class EmbeddedEventStore extends AbstractEventStore {
```

Comment:
What about adding a part to this JavaDoc about the ability to set `optimizeEventConsumption` via system properties?

Reply:
Good call @m1l4n54v1c, I've added a line of class-level javadoc for that.